sp800-63c/sec8_security.md

```diff
@@ -5,196 +5,60 @@
+| Assertion repudiation by the subscriber | Subscriber claims not to have performed transaction | User agreement (e.g., contract) cannot be enforced |
+| Assertion redirect | Assertion can be used in unintended context | Compromised user agent passes assertion to attacker who uses it elsewhere |
+| Assertion reuse | Assertion can be used more than once with same RP | Intercepted assertion used by attacker to authenticate their own session |
+| Secondary authenticator manufacture | Attacker generates a secondary authenticator to impersonate a subscriber | Attacker exploits a cryptographic weakness in generation of secondary authenticators |
+| Secondary authenticator capture | Attacker intercepts the secondary authenticator | Attacker uses a session-hijacking attack to capture the secondary authenticator |
+| Assertion substitution | Attacker uses an assertion intended for a different subscriber | Session hijacking attack between IdP and RP
+
+### 8.2. Federation Threat Mitigation Strategies
+
+Mechanisms that assist in mitigating the threats identified above are identified in Table 8-2.
+
+<div class="text-center" markdown="1">
+
+**Table 8-2. Mitigating Authenticator Threats**
+
+</div>
+
+| **Federation Threat/Attack** | **Threat Mitigation Mechanisms** |
+|------------------------------|----------------------------------|
+| Assertion Manufacture or Modification | Cryptographically sign the assertion at IdP and verify at RP |
+| | Send assertion over an authenticated protected channel authenticating the IdP |
+| | Include a non-guessable random identifier in the assertion |
+| Assertion disclosure | Send assertion over an authenticated protected channel authenticating the RP |
+| | Encrypt assertion for a specific RP (may be accomplished by use of a mutually authenticated protected channel) |
+| Assertion repudiation by the IdP | Cryptographically sign the assertion at the IdP with a key that supports non-repudiation; verify signature at RP |
+| Assertion repudiation by the subscriber | Issue holder-of-key assertions; proof of possession of presented key verifies subscriber's participation |
+| Assertion redirect | Include identity of the RP for which the assertion is issued in its signed content; RP verifies that they are intended recipient |
+| Assertion reuse | Include a timestamp with short validity period in the signed content of the assertion; RP verifies validity |
+| | RP keeps track of assertions consumed within a configurable time window to ensure that a given assertion is not used more than once. |
+| Secondary authenticator manufacture | Ensure that secondary authenticator has sufficient entropy to resist manufacture |
+| | Include timely assertion data signed by the IdP |
+| Secondary authenticator capture | Send secondary authenticator using an authenticated protected channel establishes at primary authentication time |
```

Comment:
"establishes" -> "established"
Also, say where the channel is established -- between which parties?